### ISSUE TITLE
Add Major Mode Leader Key "Comma"

### ISSUE THREAD

**Issue by gnomesoup** [VSpaceCode/VSpaceCode#115]
## Problem

I would like to map the comma, `","`, in normal mode as a shortcut to open the major mode menu,  `<space> m`. I have not been able to find anything in the documentation that would allow me to open the whichkey menu to a specific sub-menu. As this is default behavior in Spacemacs, I would like for it to be the default in VSpaceCode as well.
## Solution

Have the `vspacecode.space` command able to accept arguments that could take me directly to the whichkey sub-menu that matches the argument. So I can do something like the following in my settings json file.

```
{
    "vim.normalModeKeyBindingsNonRecursive": [
        {
            "before": [
                ","
            ],
            "commands": [
                "vspacecode.space"
            ],
            "args": [
                "m"
            ]
        }
    ]
}
```

## Alternatives

Please let me know if I missed this feature somewhere.

## Additional context

[Spacemacs Major Mode Leader Key Documentation](https://develop.spacemacs.org/doc/DOCUMENTATION.html#major-mode-leader-key)

**Comment by stevenguh** [comment]
That's a great point. We have a way to achieve this with existing command, and we should document this somehow. The follow json is an example of your request.
```
    "vim.normalModeKeyBindingsNonRecursive": [
        {
            "before": [","],
            "commands": [
                "vspacecode.space",
                {
                    "command": "whichkey.triggerKey",
                    "args": "m"
                }
            ],
        }
]
```

In addition, since this is in the spacemacs default, we probably should include in the default/initial config commands.

**Comment by gnomesoup** [comment]
That is awesome! I found `whichkey.triggerKey` but I wasn't nesting my commands correctly. Thanks for pointing out the correct way. I would love to see this as a default.

**Comment by stevenguh** [comment]
I might still consider adding a proper api for this. This way of using trigger key still have some issues with users have delay on.

**Comment by stevenguh** [comment]
Just fixed the bugs with calling triggerKey right after show command for users with delay on with the which-key v0.8.2. I think we probably can use the above binding for now without adding a new API.

**Comment by stevenguh** [comment]
Fixed in a0060b5b8a104de994735515d74f9f2b3d18b9a0, and will be part of the next release
